compiler/rustc_data_structures/src/dyn_send_sync_impl.rs

```diff
@@ -0,0 +1,114 @@
+use crate::sync::{DynSend, DynSync};
```

Comment:
I'd move all `DynSend` and `DynSync` related things to a `marker` module.

Reply:
It makes since. Done.